CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpFormatsViewController.swift

```diff
@@ -90,9 +90,23 @@ class HelpFormatsViewController: UIViewController {
     private func configureConstraints() {
         view.addConstraints([
```

Comment:
According to Apple documentation, this should be avoided: 

When developing for iOS 8.0 or later, use the [NSLayoutConstraint](doc://com.apple.documentation/documentation/uikit/nslayoutconstraint?language=swift) class’s [activate(_:)](doc://com.apple.documentation/documentation/uikit/nslayoutconstraint/1526955-activate?language=swift) method instead of calling the addConstraints(_:) method directly. The activate(_:) method automatically adds the constraints to the correct views.